version.txt

```diff
@@ -0,0 +1 @@
+1.18.0-dev
```

Comment:
Let's try it without the version.txt first and see what happens

Reply:
Sounds good.